cmd/enforce_test.go

```diff
@@ -15,15 +15,138 @@
 package cmd
 
 import (
+	"encoding/json"
 	"testing"
+
+	"github.com/stretchr/testify/require"
 )
 
+func Test_enforceExCmd(t *testing.T) {
```

Comment:
Write test cases in Casbin flavor: https://github.com/casbin/casbin/blob/a98973beed58c2236ee453a126970f00d68c0f34/rbac_api_test.go#L232-L253